policyengine_us/variables/household/income/person/interest/tax_exempt_dividend_income.py

```diff
@@ -0,0 +1,13 @@
+from policyengine_us.model_api import *
+
+
+class tax_exempt_dividend_income(Variable):
```

Comment:
@MaxGhenis is there an existing variable we can use or does creating this work?

Reply:
@martinholmer could you advise here on how best to capture these state-level adjustments based on tax-exempt income from state and local bonds? Based on how you've been approaching it so far and what would best align to taxsim.

Reply:
What is @PavelMakarchuk reading in the VT income tax code that leads him to define a new variable called `tax_exempt_dividend_income`?

Reply:
Since we had tax_exempt_interest_income the development team has asked whether we need an equivalent for dividend income
[2022 Schedule IN-112
](https://tax.vermont.gov/sites/tax/files/documents/IN-112-2022.pdf#page=1)
![image](https://github.com/PolicyEngine/policyengine-us/assets/110687043/d1104ffe-2f26-4f14-9e02-37bac1949a5f)